addons/pom.xml

```diff
@@ -18,6 +18,7 @@
     <module>binding</module>
     <module>io</module>
     <module>ui</module>
+    <module>action</module>
```

Comment:
Is there a reason the module list is no longer sorted?
If so, please document the reason in a comment or otherwise just sort the list. :-)